Localide/Classes/UIApplication+Localide.swift

```diff
@@ -1,30 +0,0 @@
-//
-//  UIApplication+Localide.swift
-//  Localide
-//
-//  Created by David Elsonbaty on 5/29/16.
-//  Copyright © 2016 David Elsonbaty. All rights reserved.
-//
-
-import UIKit
-
-// Special thanks to @dianz (http://stackoverflow.com/a/30858591)
-internal extension UIApplication {
```

Comment:
And what we say to magic?

![image](https://user-images.githubusercontent.com/39052976/92443713-8bce8500-f1ba-11ea-944f-427be62f2340.png)

Reply:
lol